src/site/content/en/animations/animations-guide/index.md

```diff
@@ -105,52 +71,51 @@ you could use `transform` and all of the work could be done in the composite ste
   }
 }
 ```
+{% endCompare %}
 
-[See this example on Glitch](https://glitch.com/~animation-with-transform).
+## Resize an element {: #resize }
 
-Test this example in Chrome and Firefox DevTools to see the difference that the change makes.
+To resize an element, use the `scale` keyword value of the
+[`transform`](https://developer.mozilla.org/en-US/docs/Web/CSS/transform) property.
 
-## Finding paint problems
+## Change an element's visibility {: #visibility }
 
-DevTools can help you to identify which parts of your page are being repainted.
-In Chrome DevTools, open the Rendering Tab and select Paint Flashing.
-In Firefox DevTools add a Toolbox button for [Paint Flashing](https://developer.mozilla.org/en-US/docs/Tools/Paint_Flashing_Tool).
-These tools will highlight areas of the page that will be repainted.
+To show or hide an element, use [`opacity`](https://developer.mozilla.org/en-US/docs/Web/CSS/opacity).
 
-<figure class="w-figure">
-  <img class="w-screenshot" src="./paint-flashing.jpg" alt="A UI element highlighted in green to demonstrate it will be repainted">
-</figure>
+<!-- TODO(kaycebasques): Add basic examples -->
 
-If you see the whole screen flashing,
-or areas that you don't think should change highlighted then you can do some investigation.
+## Avoid properties that trigger layout or paint {: #triggers }
```

Comment:
Another example of rephrasing the section heading to focus on the task that the reader should do (in this case we're advising them to avoid something they may not think about whereas with the "move an element" section heading we are describing the use case that they care about)